test/ares-test.cc

```diff
@@ -745,6 +745,14 @@ MockChannelOptsTest::MockChannelOptsTest(int count,
     opts.tries = 3;
     optmask |= ARES_OPT_TRIES;
   }
+  // If not already overridden, set server failover options. For reliable
+  // testing we set the retry chance to 100%, and the retry delay to a high
+  // value (1 hour) so that retries are never performed on a test.
+  if (!(optmask & ARES_OPT_SERVER_FAILOVER)) {
+    opts.server_failover_opts.retry_chance = 1;
+    opts.server_failover_opts.retry_delay = 60 * 60 * 1000;
+    optmask |= ARES_OPT_SERVER_FAILOVER;
+  }
```

Comment:
So the default retry delay is 5000ms, and some tests currently take ~1500ms to run.

I was thinking to leave this in, just in case longer-running tests are added in the future that hit the 5000ms default. Although it is probably safe to remove!

Reply:
i'd go ahead and remove it since all tests pass at a much lower timeout.  besides retry_chance of 1 just seems wrong.

Reply:
This is reverted now.